docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+`#` characters:
+
+```bash
+######
+export HUSKY_LOGITECH=1
+```
+
+Save the file and quit nano.  Then restart ROS by running `sudo systemctl restart ros` or rebooting the robot.
+When ROS restarts it will now use the Logitech controller as its teleoperation input device.
+
+### Setting up Husky's Network Configuration    {#husky-networking}
+
+Husky is normally equipped with a combination Wifi + Bluetooth module.
+If this is your first unboxing, ensure that Husky's wireless antennas are firmly screwed on to the chassis.
+Some Husky robots may only be equipped with a single antenna, depending on the exact model of PC installed in the robot.
+
+#### First Connection
+
+By default, Husky's wifi is in client mode, looking for the wireless network at the Clearpath factory.
+
+<ComponentWiredRobotConnection />
+
+#### Changing the Default Password
+
+<ComponentChangingDefaultPassword />
+
+#### Wifi Setup
+
+<ComponentWifiRobotConnection />
+
+### Installing Remote PC Software    {#remote-pc-software}
+
+:::note
+This step is optional.
+:::
+
+It is often convenient to use a Remote PC (eg. laptop) to command and observe your robot. To do this, your
+Remote PC must be configured correctly.
+
+1. Perform a basic ROS installation. See [here](http://wiki.ros.org/noetic/Installation/Ubuntu) for details.
+
+2. Install the Husky desktop packages:
+    ```bash
+    sudo apt-get install ros-noetic-husky-desktop
+    ```
+
+3. Configure Remote ROS Connectivity.
+
+    <details><summary>Click to expand</summary><p>
+    <ComponentRemoteRosConnectivity />
+    </p></details>
+
+4. From your Remote PC, try launching rviz, the standard ROS robot visualization tool:
+
+    ```bash
+    roslaunch husky_viz view_robot.launch
+    ```
+
+    From within rviz, you can use interactive markers to drive Husky, you can visualize its published
+    localization estimate and you can visualize any attached sensors which have been added to its robot
+    description XML _URDF_.
+
+### Customizing Husky Configuration
+
+.. Note:: These tutorials assume that you are familiar with ROS and the catkin build system.
+Please familiarize yourself using the [ROS](http://wiki.ros.org/ROS/Tutorials) and
+[catkin](http://wiki.ros.org/catkin/Tutorials) tutorials.
+
+If upgrading from a prior ROS release, you should now re-examine your backed-up files from
+[Performing a Backup](#performing-a-backup) to determine if there's any customizations that
+need to be configured on your platform.
+
+#### Environment Variables    {#environment-variables}
+
+Husky can be customized and extended through the use of several environment variables. These are summarized in the
+[README](https://github.com/husky/husky/blob/noetic-devel/husky_description/README.md) file.
+
+#### Adding a Source Workspace
+
+Configuring non-standard peripherals requires a source workspace on the robot PC.
+
+1. Create a new workspace:
+
+    ```bash
+    mkdir -p ~/husky_noetic_ws/src
```

Comment:
Consider adding an Admonition Tip that husky_noetic_ws can be changed to whatever the user wants.
(Unless you plan to cover this in the updated ROS tutorials)

Reply:
Done